ui/segmented-bar.reel/segmented-bar.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * @module montage/ui/segmented-bar.reel
+ * @requires montage/ui/base/abstract-button.AbstractButton
+ * @requires montage/core/tree-controller.TreeController
+ */
+var AbstractButton = require("ui/base/abstract-button").AbstractButton;
+
+/**
+ * @class SegmentedBar
+ * @classdesc
+ * A segmented bar.
+ * Reference design: OS X storage space visualization found in iTunes or About This Mac
+ * Inheriting from AbstractButton because SegmentedBar needs its PressComposer behavior
+ *
+ * @extends AbstractButton
+ *
+ * @see {@link https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Flexible_boxes}
+ * for FlexBox usage
+ */
+exports.SegmentedBar = AbstractButton.specialize( /** @lends SegmentedBar.prototype # */ {
```

Comment:
> Declaring the _content, _contentController, and _orientation properties is unnecessarily verbose. Those private properties are only used in the content, contentController, and orientation setters and getters and they don't need to be declared outside of those. As an added bonus, keeping _content, _contentController, and _orientation to within their corresponding accessors will ensure those properties are created only if they are needed.

I see that everything works w/o those private variables. Why is it all over the place in the codebase? It seems to be an odd convention...